src/sidebar/util/test/fetch-config-test.js

```diff
@@ -147,17 +146,95 @@ describe('sidebar.util.fetch-config', () => {
             })
           );
 
-        return fetchConfig({}, fakeWindow).then(config => {
-          assert.deepEqual(config, {
-            apiUrl: 'https://servi.ce/api/',
-            services: [
-              {
-                apiUrl: 'https://servi.ce/api/',
-                grantToken: 'secret-token',
+        const config = await fetchConfig({}, fakeWindow);
+        assert.deepEqual(config, {
+          apiUrl: fakeApiUrl(),
+          services: [
+            {
+              apiUrl: 'https://servi.ce/api/',
+              grantToken: 'secret-token',
+            },
+          ],
+        });
+      });
+    });
+
+    context('from an RPC parent of known ancestry', () => {
+      // `requestConfigFromFrame` is an object containing an `origin` and `ancestorLevel`
+      //
+      // In scenarios like LMS integrations, the client is annotating a document
+      // inside an iframe and the client needs to retrieve configuration
+      // securely from the top-level window without  that configuration being
+      // exposed to the document itself.
+      beforeEach(() => {
+        fakeJsonRpc.call.resolves({});
+        fakeHostConfig.returns({
+          requestConfigFromFrame: {
+            origin: 'https://embedder.com',
+            ancestorLevel: 2,
+          },
+        });
+      });
+      describe('getAncestorFrame', () => {
```

Comment:
`getAncestorFrame` is an implementation detail rather than an exported API method so it doesn’t need to be mentioned in the tests.

Reply:
awe.. that was left over from refactoring, forgot to remove it!